mne/io/base.py

```diff
@@ -1997,6 +2004,16 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
 
         pos_prev = pos
 
+    if raw.annotations is not None:
+        start_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MIN, raw.annotations.onset)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MAX, raw.annotations.duration)
+        write_name_list(fid, FIFF.FIFF_COMMENT, raw.annotations.description,
+                        separator=';')
```

Comment:
If you really want, you could say that they aren't allowed to use semicolons, then swap instances of colons with semicolons when writing. On read you do the reverse swap.